src/app/layout.tsx

```diff
@@ -23,25 +23,21 @@ export const metadata = {
   description: "CTO at @Rocketseat. Passionate about education and changing people's lives through programming.",
 }
 
-export default function RootLayout({
-  children,
-}: {
-  children: React.ReactNode
-}) {
+export default function RootLayout({ children }: { children: React.ReactNode }) {
   return (
     <html lang="en" className={inter.className}>
       <head />
-      <body className="bg-[#7F7FD5] bg-app">
-        <div className="z-10 relative h-screen p-20 flex items-center justify-center">
-          <div className="bg-[#232135] overflow-hidden border border-[#72707D] w-full max-w-[1480px] aspect-video shadow-md shadow-black/20 rounded-lg grid grid-rows-layout">
+      <body className="w-screen h-[100svh] bg-[#7F7FD5] bg-app overflow-hidden">
```

Comment:
I never saw `svh` being used, can you tell me whats for?

Reply:
with conventional **VH**, the footer is being hidden on mobile. This is happening because the page is creating a scroll bar, which, I think, should not be happening. Please correct me if I'm wrong.

![](https://res.cloudinary.com/dmu4jj4si/image/upload/v1677427956/Screenshot_1_egnwgh.png)

The 'SVH' occupies the maximum available space respecting the navigation bar in mobile browsers,
<img src="https://web-dev.imgix.net/image/vS06HQ1YTsbMKSFTIPl2iogUQP73/KtZ9bXwtzmo7hguwQOWO.png?auto=format&w=845" width="450" height="350" />
but then I researched more about it and found that this measure is not yet supported for all browsers

the solution I found was this  `<body className="fixed h-full" ></body>` or simply let the page scroll. What do you think ?

Reply:
Oh, got it! Have you tried using height 100% inside html, body and all containers around the application?

Reply:
ok, it worked.